src/init.c

```diff
@@ -522,6 +527,8 @@ static void mi_process_done(void) {
   process_done = true;
 
   #if defined(_WIN32) && !defined(MI_SHARED_LIB)
+  // Explicitly clean up main thread
+  _mi_thread_done(_mi_heap_default);
```

Comment:
I think this should not call _mi_thread_done as it is from the main thread. (before, line 525 prevented this by setting the fls value to null)

Reply:
But what confuses me is that you merged the PR nevertheless...
Should I make a follow-up that addresses this?

Reply:
Ah, I corrected it in another later commit :-)
The latest `dev` (and `dev-slice`) should be good.